File: services/python/flask/src/routes/handlers/invalid_method.py
import structlog
from flask import request
from werkzeug.exceptions import HTTPException
from routes.errors import AppErrors, send_error


def invalid_method_handler(_error: HTTPException):
    structlog.getLogger().warn(
        "Invalid method", method=request.method, path=request.path
    )
    return send_error(AppErrors.InvalidMethod)

File: services/go/chi/routes/handlers/invalid_method.go
package handlers

import (
	"chi/routes"
	"log/slog"
	"net/http"
)

func InvalidMethodHandler(w http.ResponseWriter, r *http.Request) {
	slog.Warn("Invalid method", "method", r.Method, "path", r.URL.Path)

	routes.AppErrors.InvalidMethod.Error(w)
}
